### PR TITLE
Robolimb Durability Buff

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Instead of holy shit ten damage what the fuck, robolimbs now break at around the same time as human limbs. However, more fuel and cable is required to fix them.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -1097,7 +1097,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	return ..() && !is_dislocated() && !(status & ORGAN_TENDON_CUT) && (!can_feel_pain() || get_pain() < pain_disability_threshold) && brute_ratio < 1 && burn_ratio < 1
 
 /obj/item/organ/external/proc/is_malfunctioning()
-	return ((status & ORGAN_ROBOT) && (brute_dam + burn_dam) >= 10 && prob(brute_dam + burn_dam))
+	return ((status & ORGAN_ROBOT) && (brute_dam + burn_dam) >= 30 && prob(brute_dam + burn_dam))
```
Comment (NonQueueingMatt) [inline code/modules/organs/organ_external.dm]: Terrible way to do it. Don't make a hard check.
```suggestion
	return (BP_IS_ROBOTIC(src) && (brute_ratio + burn_ratio) >= 0.3 && prob(brute_dam + burn_dam))
```
```

File: code/modules/organs/organ_external.dm [hunk 2]
```diff
@@ -1097,7 +1097,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	return ..() && !is_dislocated() && !(status & ORGAN_TENDON_CUT) && (!can_feel_pain() || get_pain() < pain_disability_threshold) && brute_ratio < 1 && burn_ratio < 1
 
 /obj/item/organ/external/proc/is_malfunctioning()
-	return ((status & ORGAN_ROBOT) && (brute_dam + burn_dam) >= 10 && prob(brute_dam + burn_dam))
+	return (BP_IS_ROBOTIC(src) && (brute_ratio + burn_ratio) >= 0.3) && prob(brute_dam + burn_dam)
```
Comment (NonQueueingMatt) [inline code/modules/organs/organ_external.dm]:
```suggestion
	return (BP_IS_ROBOTIC(src) && (brute_ratio + burn_ratio) >= 0.3 && prob(brute_dam + burn_dam)
```

Comment (NonQueueingMatt) [inline code/modules/organs/organ_external.dm]:
```suggestion
	return (BP_IS_ROBOTIC(src) && (brute_ratio + burn_ratio) >= 0.3 && prob(brute_dam + burn_dam))
```

**Pull request merged**